### ISSUE TITLE
Socket file

### ISSUE THREAD

**Issue by pizzafox** [Dev-Yukine/Kurasuta#96]
This problem keeps occurring and causing cluster spawning to fail. I'm running the latest version of Kurasuta off of NPM. The solution is to compose down and then compose up afterward.

```js
[3/16/2019] [3:59:19 PM] [shard manager] › ✖  fatal
     An error occured when spawning clusters
[3/16/2019] [3:59:19 PM] [shard manager] › ✖  error
     Error: connect ECONNREFUSED /tmp/DiscordBot.sock
```

**Comment by kyranet** [comment]
Alternatively, you can try to use a port instead of a socket file, it's more reliable as it doesn't rely on that file (which can not be deleted, for example when the master process shuts down without closing the socket).

You can achieve by setting `ipcSocket: 9999`, or remove the option altogether, as that value is the default.

**Comment by pizzafox** [comment]
The NPM version of Kurasuta has the default `ipcSocket` set to
```js
os_1.platform() === 'win32' ? '//./pipe/tmp/DiscordBot.sock' : '/tmp/DiscordBot.sock'
```
The latest GitHub version defaults to HTTP. Should I be using that instead?

**Comment by kyranet** [comment]
Oh! I thought the NPM version was defaulting to HTTP, my bad!

And yeah, you should be good with a port number 😄

**Comment by Dev-Yukine** [comment]
Oh i see, i actuall forgot to update the npm release to the latest change im sorry for that, i will update deps to latest and then release an update on npm 👍

**Comment by Dev-Yukine** [comment]
Released kurasuta version 0.2.17 🎉
